### PR TITLE
Miljo/3.0.0/various cleanup

### PR DESCRIPTION
**Pull request by battletoilet**

### CODE REVIEW

**Comment by mikewheaton** [comment]
Approved

File: src/documentation/pages/CommandButton/CommandButton.md [hunk 1]
```diff
@@ -1,10 +1,97 @@
...
-<!--- {{> CommandButton props=propsInline}} --->
-<!--- {{> CommandButton props=propsInline}} --->
-<!--- {{> CommandButton props=propsNoLabel}} --->
-<!--- {{> CommandButton props=propsNoLabelSplit}} --->
-<!--- {{> CommandButton props=propsPivot}} --->
-<!--- {{> CommandButton props=propsSplit}} --->
-<!--- {{> CommandButton props=propsTextOnly}} --->
\ No newline at end of file
+# CommandButton
+Allows users to take action, such as submitting a form or dismissing a message. Primary use is in command bar or in Callout.
```
Comment (Jahnp) [inline src/documentation/pages/CommandButton/CommandButton.md]: Small nit: "command bar" -> "CommandBar"

File: src/utilities/Animate.ts [hunk 2]
```diff
@@ -41,8 +41,31 @@ namespace fabric {
...
+      "scaleY", 
+      "scaleZ", 
+      "rotate", 
+      "rotateX", 
+      "rotateY", 
+      "rotateZ", 
+      "skewX", 
+      "skewY"
+    ];
+	  private static _filters: Array<string> = [
```
Comment (Jahnp) [inline src/utilities/Animate.ts]: The tabbing for these two arrays could be normalized and aligned w/the statics below

**Comment by Jahnp** [comment]
Looks great! Approved

**Pull request merged**